portfolio/src/main/webapp/style.css

```diff
@@ -44,6 +44,16 @@ body {
     font-family: 'Roboto Mono', monospace; 
 }
 
+/* style h2 */
```

Comment:
i don't think this (and the next) comment is necessary

also you can combine them:
```
h2,
h4 {
    font-weight: normal;
}
```